cmd/kubeadm/app/util/chroot_windows.go

```diff
@@ -0,0 +1,34 @@
+// +build windows
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+)
+
+// MaybeChroot chroot()s to the new path, if non-empty.
+// NB: All file paths after this call are effectively relative to
+// `rootfs`
+func MaybeChroot(rootfs string) error {
+	if rootfs == "" {
+		return nil
+	}
+
+	return fmt.Errorf("Chroot feature is unimplemented for windows")
```

Comment:
`Chroot feature is unimplemented for windows`
->
`chroot is not implemented on Windows`

Reply:
(done)